code/modules/food_and_drink/snacks.dm

```diff
@@ -807,12 +807,12 @@ ABSTRACT_TYPE(/obj/item/reagent_containers/food/snacks/soup)
 	prize = 0
 
 /obj/item/reagent_containers/food/snacks/cereal_box/cirrial
-	name = "cereal box -'Flocked-Flakes Cirrial'"
-	desc = "A bluey-green cereal that beeps gently at you, they're grrrrowing out of the box oh fuck!"
-	icon_state = "cereal_box5"
-	initial_volume = 20
-	initial_reagents = list("feather_fluid"=10)
-	prize = 0
+    name = "cereal box -'Flocked-Flakes Cirrial'"
+    desc = "A bluey-green cereal that beeps gently at you, they're grrrrowing out of the box, oh fuck!"
+    icon_state = "cereal_box6"
+    initial_volume = 20
+    initial_reagents = list("flockdrone_fluid"=20)
```

Comment:
This stuff evaporates if it's not in a gnesis container

Reply:
Removed the chemical from the cereal for now, may rework its contents in the future.